scrapy/utils/reqser.py

```diff
@@ -47,7 +50,11 @@ def request_from_dict(d, spider=None):
     eb = d['errback']
     if eb and spider:
         eb = _get_method(spider, eb)
-    return Request(
+    try:
```

Comment:
The `load_object` call could fail as well. Imagine you have a disk queue containing [`scrapy_splash.request.SplashRequest`](https://github.com/scrapy-plugins/scrapy-splash/blob/master/scrapy_splash/request.py#L14) and you try to read the queue from an enviroment which doesn't have `scrapy_splash` installed.
I'm not saying how often that would happen, just that it _could_ happen :-)